packages/react-ai/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "@aws-amplify/typescript-config/tsconfig.react.json",
-  "include": ["jest.*.ts", "src/**/*.ts", "src/**/*.tsx"],
+  "include": ["jest.*.ts", "src/**/*.ts", "src/**/*.tsx", "__tests__/exports.ts"],
```

Comment:
Can we move the exports test directory inside _src_ to avoid having a snowflakey _tsconfig.json_ for the ai package?